src/vcVersion.h

```diff
@@ -14,8 +14,8 @@
 #endif
 
 // This is the standard version number string [Major].[Minor].[Revision].[BuildID]
```

Comment:
This doesn't really apply anymore? It looks like we've gone to `[Major].[Minor].[BuildID]` now?